### ISSUE TITLE
Add icons with 2x resolution for hidpi screens

### ISSUE THREAD

**Issue by guillep** [pharo-project/pharo-icon-packs#6]
With the current icons, scaling makes them blurry. We will need 2xsized icons to scale 2x.

**Comment by astares** [comment]
So 32x32 instead of 16x16?

**Comment by guillep** [comment]
That is what our expert @ronsaldo suggested

**Comment by PharoProject** [comment]
That is what the expert @ronsaldo said :)

**Comment by astares** [comment]
Yes. I already thought about having additional "light" vs. "dark" pharo icon themes, because for some themes they do not look good in dark. And we have to check since most of the icons are only available in small size (16x16) without having a larger version to scale them down.

From the graphics tools I have knowledge and can help. But question is if we have a good base for an icon set in that size.
